src/client/sky.cpp

```diff
@@ -158,6 +197,7 @@ void Sky::render()
 
 		const f32 t = 1.0f;
 		const f32 o = 0.0f;
+		const bool has_tex = m_sky_params.textures.size() == 6;
```

Comment:
wouldn't it be cleaner if this was based on the skybox type, maybe with a warning logged (once) if the wrong amount of textures is supplied?